Formula/kedge.rb

```diff
@@ -0,0 +1,27 @@
+class Kedge < Formula
+  desc "Simple, Concise & Declarative Kubernetes Applications"
+  homepage "http://kedgeproject.org"
+  url "https://github.com/kedgeproject/kedge/archive/v0.4.0.tar.gz"
+  sha256 "383a34db1ec8faf65ec8927f1e3041998c2a414a30191339c4a39385f8717475"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkpath buildpath/"src/github.com/kedgeproject"
```

Comment:
It's a teeny tiny thing but `(buildpath/"src/github.com/kedgeproject").mkpath` is the standard style.

Reply:
thx, updated